index.html

```diff
@@ -14,14 +14,20 @@
     <head>
         <nav>
```

Comment:
[OPTIONAL]
- [ ] Kindly note that the `head` tag is used before the `body` as you did and it includes hidden information about your website, the `head` tag you used which wraps the `nav` should be named `header` instead, please fix it.